llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -3372,17 +3372,12 @@ MachineBasicBlock *AArch64TargetLowering::EmitInstrWithCustomInserter(
   case AArch64::F128CSEL:
     return EmitF128CSEL(MI, BB);
   case AArch64::I32CTSELECT:
```

Comment:
You don't need to check for these condition if custom inserter is disabled in instruction definition.